hw/ip/ascon/data/ascon.hjson

```diff
@@ -0,0 +1,696 @@
+          '''
+        }
+        { bits: "2",
+          name: "WAIT_EDN",
+          desc:  '''
+            The Ascon unit is waiting for new entropy.
+          '''
+        }
+        { bits: "3",
+          name: "ASCON_ERROR",
+          desc:  '''
+            An error due to a misconfiguration has happened.
+            The user should read out the error register for more information
+          '''
+        }
+        { bits: "4",
+          name: "ALERT_RECOV_CTRL_UPDATE_ERR",
+          resval: "0",
+          desc:  '''
+            An update error has not occurred (0) or has occurred (1) in the shadowed Control Register.
+            Ascon operation needs to be restarted by re-writing the Control Register.
+          '''
+        }
+        { bits: "5",
+          name: "ALERT_RECOV_CTRL_AUX_UPDATE_ERR",
+          resval: "0",
+          desc:  '''
+            An update error has not occurred (0) or has occurred (1) in the shadowed Auxiliary Control Register.
+            The register has to be rewritten.
+          '''
+        }
+        { bits: "6",
+          name: "ALERT_RECOV_BLOCK_CTRL_UPDATE_ERR",
+          resval: "0",
+          desc:  '''
+            An update error has not occurred (0) or has occurred (1) in the shadowed block Control Register.
+            The register has to be rewritten.
+          '''
+        }
+        { bits: "7",
+          name: "ALERT_FATAL_FAULT",
+          resval: "0",
+          desc:  '''
+            No fatal fault has occurred inside the Asconunit (0).
+            A fatal fault has occurred and the Ascon unit needs to be reset (1).
+            Examples for fatal faults include
+            i) storage errors in the Control Register,
+            ii) if any internal FSM enters an invalid state,
+            iii) if any sparsely encoded signal takes on an invalid value,
+            iv) errors in the internal round counter,
+            v) escalations triggered by the life cycle controller, and
+            vi) fatal integrity failures on the TL-UL bus.
+          '''
+        }
+      ]
+    },
+    { name: "OUTPUT_VALID",
+      desc: '''
+        Output Valid Register
+        This register specifies which output register contains valid data and should be read next.
+        It also contains the status information whether the TAG comparison was valid or not.
+      ''',
+      swaccess: "ro",
+      hwaccess: "hwo",
+      fields: [
+        { bits: "2:0",
+          name: "DATA_TYPE",
+          desc:  '''
+            Specifies which output type/register is valid. There are:
+            PT_OUT, CT_OUT, TAG_OUT, NONE
+          '''
+        }
+        { bits: "4:3",
+          name: "TAG_COMPARISON_VALID",
+          desc:  '''
+            Indicates if the tag could be successfully compared 2'b01, or not 2'b10
+            2'b00 indicates that the tag hasn't been calculated, yet
+            2'b11 is invalid.
+          '''
+        }
+      ]
+    },
+    { name: "FSM_STATE",
```

Comment:
CSRNG, ENTROPY_SRC and EDN have this. And I think it's very useful for debugging. Especially since ASCON is a lot more complex than e.g. AES and has state.